forc/src/ops/forc_run.rs

```diff
@@ -185,3 +185,50 @@ fn create_tx_with_script_and_data(
 fn format_hex_data(data: &str) -> &str {
     data.strip_prefix("0x").unwrap_or(data)
 }
+
+fn try_parse_bytes32(raw: String) -> Result<fuel_tx::Bytes32, String> {
```

Comment:
Why not just use `Bytes32::from_str`? I think it basically does the same thing as this function.

Reply:
> Why not just use `Bytes32::from_str`?

Because I didn't realize I could use it 😂